### PR TITLE
Update reference.xml - add example for unix socket

### PR DESCRIPTION
**Pull request by wapmorgan**

### CODE REVIEW

File: reference/pdo_pgsql/reference.xml [hunk 1]
```diff
@@ -132,6 +132,12 @@
        a PostgreSQL database:
        <programlisting><![CDATA[
 pgsql:host=localhost;port=5432;dbname=testdb;user=bruce;password=mypass
+]]>
+       </programlisting>
+       The following example shows a PDO_PGSQL DSN for connecting to
+       a PostgreSQL database via unix socket /tmp/.s.PGSQL.5432:
+       <programlisting><![CDATA[
+pgsql:host=/tmp;port=5432;dbname=testdb;user=bruce;password=mypass
 ]]>
```
Comment (Girgias) [inline reference/pdo_pgsql/reference.xml:141]: I know it was already pressent but can you fix the markup at the same time:
```suggestion
       <programlisting>
<![CDATA[
pgsql:host=localhost;port=5432;dbname=testdb;user=bruce;password=mypass
]]>
       </programlisting>
       The following example shows a PDO_PGSQL DSN for connecting to
       a PostgreSQL database via unix socket /tmp/.s.PGSQL.5432:
       <programlisting>
<![CDATA[
pgsql:host=/tmp;port=5432;dbname=testdb;user=bruce;password=mypass
]]>
```

File: reference/pdo_pgsql/reference.xml [hunk 2]
```diff
@@ -130,8 +130,16 @@
       <para>
        The following example shows a PDO_PGSQL DSN for connecting to
        a PostgreSQL database:
-       <programlisting><![CDATA[
+       <programlisting>
+<![CDATA[
 pgsql:host=localhost;port=5432;dbname=testdb;user=bruce;password=mypass
+]]>
+       </programlisting>
+       The following example shows a PDO_PGSQL DSN for connecting to
```
Comment (cmb69) [inline reference/pdo_pgsql/reference.xml:138]: To somewhat separate the examples, we should at least have:
```suggestion
       </programlisting>
      </para>
      <para> 
       The following example shows a PDO_PGSQL DSN for connecting to
```
Alternatively, we could actually split the examples, but that might be overkill.

File: reference/pdo_pgsql/reference.xml [hunk 3]
```diff
@@ -130,8 +130,16 @@
...
+       a PostgreSQL database via unix socket /tmp/.s.PGSQL.5432:
```
Comment (cmb69) [inline reference/pdo_pgsql/reference.xml:139]: The socket name should be marked up:
```suggestion
       a PostgreSQL database via unix socket <filename>/tmp/.s.PGSQL.5432</filename>:
```

**Comment by cmb69** [comment]
Thank you!

**Pull request closed without merging**